<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="buffer_layout">
       <item>
        <layout class="QHBoxLayout" name="buffer_slider_layout">
         <item>
          <widget class="QLabel" name="buffer_label">
           <property name="text">
            <string>Buffer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="buffer_slider">
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="buffer_lineEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="fps_layout_2">
       <item>
        <layout class="QHBoxLayout" name="fps_slider_layout">
         <item>
          <widget class="QLabel" name="fps_label">
           <property name="text">
            <string>FPS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="fps_slider">
           <property name="maximum">
            <number>120</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="fps_lineEdit">
         <property name="text">
          <string>60</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignBottom">
        <widget class="QPushButton" name="start_button">
         <property name="text">
          <string>Start</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="dim_layout">
       <item>
        <layout class="QHBoxLayout" name="height_layout">
         <item>
          <widget class="QLabel" name="height_label">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="height_lineEdit">
           <property name="text">
            <string>720</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="width_layout">
         <item>
          <widget class="QLabel" name="width_label">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="width_lineEdit">
           <property name="text">
            <string>1200</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
